safetrace-app/AppDelegate.swift

```diff
@@ -88,3 +93,19 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
         environment.safeTrace.session.setAPNSToken(deviceToken)
     }
 }
+
+// MARK: - App Switcher Overlay
+extension AppDelegate {
+    private func showAppSwitcherOverlay() {
+        appSwitcherOverlayWindow = UIWindow()
+        appSwitcherOverlayWindow?.rootViewController = AppSwitcherOverlayViewController()
+        appSwitcherOverlayWindow?.windowLevel = .alert + 1
+        appSwitcherOverlayWindow?.backgroundColor = .clear
+        appSwitcherOverlayWindow?.makeKeyAndVisible()
+    }
+
+    private func hideAppSwitcherOverlay() {
+        appSwitcherOverlayWindow?.isHidden = true
```

Comment:
Should we set the rootViewController back to `MainNavigationController` here?

Reply:
The main window's rootViewController was never changed — we're just getting rid of the overlay window

Reply:
Ahh I misread. Yeah this makes sense